gemviz/bluesky_runs_catalog.py

```diff
@@ -78,7 +78,7 @@ def catalog(self):
     def catalogName(self):
         return self.parent.catalogName()
 
-    def splitter_moved(self, key):
+    def splitter_moved(self, key,*arg,**kwargs):
```

Comment:
please black code style to this line (spaces after the commas)

Reply:
done